packages/grafana-ui/src/components/Menu/MenuGroup.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useStyles } from '../../themes';
+import { MenuItemProps } from './MenuItem';
+
+/** @internal */
+export interface MenuItemsGroup {
+  /** Label for the menu items group */
+  label?: string;
+  /** Items of the group */
+  items: MenuItemProps[];
```

Comment:
seems like you don't use this propert as you use MenuItem to render a single item :)

Reply:
This property is actually being used in the ContextMenu component.
Given our next line of action in ContextMenu, I think this would no longer be the case.